pkg/apis/featurestates/v1alpha1/types.go

```diff
@@ -0,0 +1,41 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CnsCsiSvFeatureStatesSpec defines the desired state of CnsCsiSvFeatureStates
+type CnsCsiSvFeatureStatesSpec struct {
+	FeatureStates []FeatureState `json:"CnsCsiSvFeatureStates,omitempty"`
+}
+
+// FeatureState defines the feature name and its state
+type FeatureState struct {
+	Name  string `json:"Name"`
```

Comment:
Can you add docs to the field in this struct? We should call out that the Name is the unique identifier of the feature and it must be unique.

Reply:
done.